src/domains/citizen-auth-app/01_network.tf

```
@@ -137,6 +142,49 @@ module "session_manager_snet" {
   }
 }
 
+resource "azurerm_private_endpoint" "session_manager_sites" {
+  name                = "${local.common_project}-session-manager-app-pep-01"
+  location            = var.location
+  resource_group_name = azurerm_resource_group.session_manager_rg_weu.name
+  subnet_id           = data.azurerm_subnet.private_endpoints_subnet.id
+
+  private_service_connection {
+    name                           = "${local.common_project}-session-manager-app-pep-01"
+    private_connection_resource_id = module.session_manager_weu.id
+    is_manual_connection           = false
+    subresource_names              = ["sites"]
+  }
+
+  private_dns_zone_group {
+    name                 = "private-dns-zone-group"
+    private_dns_zone_ids = [data.azurerm_private_dns_zone.privatelink_azurewebsites_net.id]
+  }
+
+  tags = var.tags
+}
+
+resource "azurerm_private_endpoint" "staging_session_manager_sites" {
+  name                = "${local.common_project}-session-manager-staging-app-pep-01"
+  location            = var.location
+  resource_group_name = azurerm_resource_group.session_manager_rg_weu.name
+  subnet_id           = data.azurerm_subnet.private_endpoints_subnet.id
+
+  private_service_connection {
+    name                           = "${local.common_project}-session-manager-staging-app-pep-01"
+    private_connection_resource_id = module.session_manager_weu_staging.id
+    is_manual_connection           = false
+    subresource_names              = ["sites-staging"]
```

What if we concat the module value instead of the hardcoded version? 
```suggestion
    subresource_names              = ["sites-${module.session_manager_weu_staging.name}"]
```